libs/csm/rest/csm_rest_quota.py

```diff
@@ -231,6 +230,57 @@ def get_user_capacity_usage(self, resource, uid,
         self.log.info("Get user quota request successfully sent...")
         return response
 
+    def verify_user_capacity(self, user_id, used:int, used_rounded:int=None, obj_cnt:int=1,
+                            eresponse=HTTPStatus.OK, resource="user"):
+        """
+        Verify user capacity
+        """
+        if used_rounded is None:
+            used_rounded = used
+        resp = self.get_user_capacity_usage(resource, user_id)
+        result =  resp.status_code == eresponse
+        if not result:
+            self.log.error("Status code check failed for get capacity")
+        else:
+            user_details = resp.json()["capacity"]["s3"]["users"][0]
+            auser_id = user_details["id"]
+            self.log.info("Actual: %s", auser_id)
+            self.log.info("Expected: %s", user_id)
+
+            if user_id != auser_id:
+                self.log.info("User ID mismatch")
+                #result = False
```

Comment:
please remove the commented code if not required

Reply:
Uncommented the code.